gradle/libs.versions.toml

```diff
@@ -36,3 +36,4 @@ jetbrains-compose = { id = "org.jetbrains.compose", version.ref = "jetbrains-com
 ben-manes-versions = "com.github.ben-manes.versions:0.51.0"
 nexus-publish = "io.github.gradle-nexus.publish-plugin:2.0.0"
 kotlinter = "org.jmailen.kotlinter:4.4.1"
+bcv = "org.jetbrains.kotlinx.binary-compatibility-validator:0.15.1"
```

Comment:
I'd like to be a bit more explicit about the name. Maybe `binary-compat-validator`?

Reply:
According to it's group, it might be `jetbrains-bcv` or `kotlinx-bcv`. Anyway, I can rename it to others you prefer.

Reply:
As you can see with the other plugins, I don't have a strict naming scheme. It should just be understandable. I like the name I proposed before 😉